telethon/client/uploads.py

```diff
@@ -53,6 +53,11 @@ def _resize_photo_if_needed(
         # Don't use a `with` block for `image`, or `file` would be closed.
         # See https://github.com/LonamiWebs/Telethon/issues/1121 for more.
         image = PIL.Image.open(file)
+        try:
+            exif = image.info['exif']
+        except KeyError:
+            exif = None
```

Comment:
If `image.info` is a `dict`, this could be simplified:
```suggestion
        exif = image.info.get('exif')
```

I'm not sure if it's a `dict` or a custom object with `__getitem__` defined though, so it would need to be tested.